packages/kbn-expandable-flyout/src/state.ts

```diff
@@ -20,8 +20,16 @@ export interface State {
   /**
    * Panels to render in the preview section
    */
-  preview: FlyoutPanelProps[];
+  preview: FlyoutPanelProps[] | undefined;
```

Comment:
as discussed, now that we are storing state in redux, consider updating the type of `FlyoutPanelProps` to be serializable ([line](https://github.com/elastic/kibana/blob/main/packages/kbn-expandable-flyout/src/types.ts#L73))

```
export interface FlyoutPanelProps {
...
/**
* Any parameters necessary for the initial requests within the flyout
*/
params?: Record<string, unknown>;
```

Reply:
yup I'm going to have to think about how to do that, I'll do it in a follow up PR!